spec.bs

```diff
@@ -7454,6 +7570,12 @@ An <dfn export>interest group</dfn> is a [=struct=] with the following [=struct/
   : <dfn>max trusted bidding signals url length</dfn>
   :: A {{long}} integer, initially 0. Indicates the maximum trusted bidding signals fetch url length
     for the interest group. 0 means no limit.
+  : <dfn>trusted bidding signals coordinator</dfn>
+  :: Null or an [=origin=], initially null. This is used to specify where to obtain the public key
+    used for encryption and decryption in communication with a trusted bidding signal server running
+    in a Trust Execution Environment (TEE). When this field is specified, the request will be sent to
+    a trusted bidding signals server running in a TEE, and the value of
+    [=interest group/trusted bidding signals url=] is ignored.

```

Comment:
you mean [max trusted bidding signals url length](https://pr-preview.s3.amazonaws.com/xtlsheep/turtledove/pull/1340.html#interest-group-max-trusted-bidding-signals-url-length)?

Reply:
Yes, my bad